jboss-kie-kieserver/tests/bats/jboss-kie-kieserver.bats

```diff
@@ -23,18 +23,73 @@ teardown() {
 @test "test if the EJB_TIMER datasource has been auto-configured" { 
   local expected_timer_service="EJB_TIMER"
   local expected_datasources="EJB_TIMER,RHPAM"
+  local expected_url="jdbc:h2:/deployments/data/h2/rhpam;AUTO_SERVER=TRUE"
   export DATASOURCES="RHPAM"
-  export RHPAM_XA_CONNECTION_PROPERTY_URL="jdbc:h2:/deployments/data/h2/rhpam;AUTO_SERVER=TRUE"
+  export RHPAM_XA_CONNECTION_PROPERTY_URL="${expected_url}"
   export RHPAM_NONXA="false"
   configure_EJB_Timer_datasource >&2
   echo "Expected EJB_TIMER url is ${EJB_TIMER_XA_CONNECTION_PROPERTY_URL}" >&2 
   echo "Expected DATASOURCES is ${DATASOURCES}" >&2 
   [ "${TIMER_SERVICE_DATA_STORE^^}" = "${expected_timer_service}" ]
-  [ "${EJB_TIMER_XA_CONNECTION_PROPERTY_URL}" = "${RHPAM_XA_CONNECTION_PROPERTY_URL}" ]
+  [ "${EJB_TIMER_XA_CONNECTION_PROPERTY_URL}" = "${expected_url}" ]
+  [ "${RHPAM_XA_CONNECTION_PROPERTY_URL}" = "${expected_url}" ]
   [ "${DATASOURCES}" = "${expected_datasources}" ]
   [ "${RHPAM_NONXA}" = "false" ]
 }
 
+@test "verify if EJB_TIMER is correctly configured with xa-data-sources when driver is mariadb" {
+  local expected_timer_service="EJB_TIMER"
+  local expected_datasources="EJB_TIMER,RHPAM"
+  export DATASOURCES="RHPAM"
+  export RHPAM_DRIVER="mariadb"
+  export RHPAM_DATABASE="rhpam-mariadb"
+  export RHPAM_USERNAME="rhpam-user"
+  export RHPAM_PASSWORD="rhpam-pwd"
+  export RHPAM_SERVICE_HOST="myapp-host"
+  export RHPAM_SERVICE_PORT="3306"
+  export RHPAM_JNDI="jboss:/datasources/rhpam"
+  export RHPAM_JTA="true"
+  configure_EJB_Timer_datasource >&2
+  echo "EJBTimer url is ${EJB_TIMER_XA_CONNECTION_PROPERTY_Url} " >&2
+  [ "${DATASOURCES}" = "${expected_datasources}" ]
+  [ "${EJB_TIMER_DRIVER}" = "${RHPAM_DRIVER}" ]
+  [ "${RHPAM_URL}" = "jdbc:mariadb://myapp-host:3306/rhpam-mariadb?enabledSslProtocolSuites=TLSv1.2" ]
+  # we do not expect that this var is set anymore, since we're using URL property directly
+  [ "${EJB_TIMER_XA_CONNECTION_PROPERTY_PinGlobalTxToPhysicalConnection}" = "" ]
+  [ "${EJB_TIMER_XA_CONNECTION_PROPERTY_EnabledSslProtocolSuites}" = "" ]
+  [ "${EJB_TIMER_XA_CONNECTION_PROPERTY_ServerName}" = "" ]
+  [ "${EJB_TIMER_XA_CONNECTION_PROPERTY_DatabaseName}" = "" ]
+  # the URL property must be set
+  [ "${EJB_TIMER_XA_CONNECTION_PROPERTY_Url}" = "jdbc:mariadb://myapp-host:3306/rhpam-mariadb?pinGlobalTxToPhysicalConnection=true\&amp;enabledSslProtocolSuites=TLSv1.2" ]
```

Comment:
changes looks good to me, but, can you add mysql tests to make sure that the pinGLobal.... prop is correctly added?
plus, add test when setting the XA properties, DatabaseName, port, PinGlobal, TLSProtos...